examples/microwave-oven-app/microwave-oven-common/src/microwave-oven-device.cpp

```diff
@@ -61,6 +94,17 @@ Protocols::InteractionModel::Status ExampleMicrowaveOvenDevice::HandleModifyCook
     return Status::Success;
 }
 
+CHIP_ERROR ExampleMicrowaveOvenDevice::GetWattSettingByIndex(uint8_t index, uint16_t & wattSetting)
+{
+    // placeholder implementation
```

Comment:
Could we expand this? Placeholder for what? Is this a future todo to enhance or are people trying to use this as an example supposed to make a "real" implementation (and if so, what is real here)?